docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -141,12 +141,12 @@ You should see output similar to the following:
 
 1. Copy `test_data.csv` and interrupt it using the following commands:
 
-    ```sh
+    ```sql
     \copy test_copy from 'test_data.csv'  (DELIMITER ',');
     ```
 
     ```output
-    ^CCancel request sent
+    Ctrl+C Cancel request sent
```

Comment:
if ^C actually shows up on the command line as shown. ^ = ctrl right?

Reply:
Yes , so do you suggest we retain it as it is or specify Ctrl?